src/sage/groups/perm_gps/permgroup_named.py

```diff
@@ -87,7 +89,9 @@
 from pathlib import Path
 
 from sage.rings.all import Integer
+from sage.interfaces.gap import gap
```

Comment:
please no more dependencies on pexpect gap. We are trying to get rid of it in #26902

Reply:
If we do the suggested self._gap_small_group = libgap.SmallGroup(order,gap_id) then we lose some functionality, needed for an application (FusionDouble, probably an upcoming PR). We need the methods conjugacy_classes_representatives and centralizer.

I haven't figured out why these do not work when we use libgap. I tried making SmallPermutationGroup inherit from PermutationGroup_generic since that class has the needed methods.

Reply:
is `FusionDouble` a new functionality, not in Sage yet?

Reply:
Yes, analogous to FusionRing one can construct the Fusion Rings of the Drinfeld Double of a finite group.